src/main/java/xyz/nucleoid/plasmid/util/BucketScanner.java

```diff
@@ -28,7 +28,7 @@ private BucketScanner() {
      * @param amount the amount of maximum blocks to find
      * @param predicate the predicate for the blocks that can be accepted in the set
```

Comment:
I would recommend documenting the purpose of the two `BlockPos` type parameters of the `BiPredicate`.

Reply:
Could alternatively use a custom functional interface here with named parameters